vms/avm/txs/executor/semantic_verifier_test.go

```diff
@@ -30,7 +31,7 @@ import (
 )
 
 func TestSemanticVerifierBaseTx(t *testing.T) {
-	ctx := newContext(t)
+	ctx := snowtest.Context(t)
```

Comment:
why don't we set `ctx.ChainID = ctx.XChainID` here, similarly to what we do with `setup` function?
Granted that this test does not need ChainID duly set, I'd feel better If I had a fully formed context. Consider passing the target chain in context constructor

Reply:
Done